src/frameworks/index.ts

```diff
@@ -234,10 +234,8 @@ function scanDependencyTree(searchingFor: string, dependencies = {}): any {
 
 export function getNodeModuleBin(name: string, cwd: string) {
   const cantFindExecutable = new FirebaseError(`Could not find the ${name} executable.`);
-  const npmBin = spawnSync("npm", ["bin"], { cwd }).stdout?.toString().trim();
-  if (!npmBin) {
-    throw cantFindExecutable;
-  }
+  const npmRoot = (_a = (0, cross_spawn_1.sync)("npm", ["root"], { cwd }).stdout) === null || _a === void 0 ? void 0 : _a.toString().trim();
```

Comment:
thanks for the fix, mind making the change to the typescript (pre-compiled) source?

Reply:
i can modify the PR if you'd rather, NBD

Reply:
Sorry, i was a bit too fast.
Is it good like this ?
Otherwise feel free to correct